sale_order_import_edifact/models/business_document_import.py

```diff
@@ -40,10 +40,9 @@ def _hook_match_partner(self, partner_dict, chatter_msg, domain, order):
             )
             raise exceptions.UserError(
                 _(
-                    "Partner GLN Code: %s not found in order file: '%s' "
-                    "from VAT registration number '%s'."
-                )
-                % (party_id, ctx.get("order_filename"), ctx.get("rff_va"))
+                    "Partner GLN Code: {} not found in order file: '{}' "
+                    "from VAT registration number '{}'."
+                ).format(party_id, ctx.get("order_filename"), ctx.get("rff_va"))
```

Comment:
no need for format anymore since https://github.com/odoo/odoo/commit/3c3b88f6768e31417263b3828a874ca204e29514

Reply:
Thank Nils, updated